lang/po/Firmware_it.po

```diff
@@ -2490,10 +2490,10 @@ msgstr ""
 "Caricamento nell' estrusore non riuscito. Ispezionare la forma della punta "
 "del fil. Affinare la calib. del sensore, se necessario."
 
-#. MSG_TITLE_MCU_UNDERVOLTAGE_VCC c=20
+#. MSG_TITLE_MCU_ERROR c=20
 #: ../../Firmware/mmu2/errors_list.h:176 ../../Firmware/mmu2/errors_list.h:222
-msgid "MMU MCU UNDERPOWER"
-msgstr "SOTTOTENSION MCU VCC"
+msgid "MMU MCU ERROR"
+msgstr "MMU MCU ERRORE"
```

Comment:
Can't say for the others, but this is better written as "ERRORE MMU MCU"

Reply:
fixed [3b9dc97](https://github.com/prusa3d/Prusa-Firmware/pull/4269/commits/3b9dc97ec673ebb4fbb30c0a67286a4dfd67b675)